core/run_ctest/main.py

```diff
@@ -58,4 +69,4 @@ def test_conf_file(conf_file_path, test_input):
 
 
 if __name__ == '__main__':
-    main()
+    main(sys.argv)
```

Comment:
@Alex-Lian can you move arg parsing to `program_input.py` make that single ctest input an optional argument? that would be great